src/Sulu/Bundle/ContentBundle/Content/Types/SmartContent/SmartContent.php

```diff
@@ -196,6 +196,22 @@ public function getViewData(PropertyInterface $property)
         $this->getContentData($property);
         $config = $property->getValue();
 
+        $config = array_merge(
+            array(
+                'dataSource' => null,
+                'includeSubFolders' => null,
+                'category' => null,
+                'tags' => null,
+                'sortBy' => null,
+                'sortMethod' => null,
+                'presentAs' => null,
+                'limitResult' => null,
+                'page' => null,
+                'hasNextPage' => null,
+            ),
+            $config
+        );
```

Comment:
ne because thats not the values which are returned.

![image](https://cloud.githubusercontent.com/assets/1464615/6284228/32f1218c-b8ee-11e4-84d4-a8ba07f555fc.png)

these values will be returned